compose/foundation/foundation/src/jsWasmMain/kotlin/androidx/compose/foundation/gestures/JsScrollable.js.kt

```diff
@@ -18,24 +18,40 @@
 
 package androidx.compose.foundation.gestures
 
+import androidx.compose.foundation.fastFold
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.input.pointer.PointerEvent
 import androidx.compose.ui.unit.Density
 import androidx.compose.ui.unit.IntSize
+import kotlin.math.sqrt
 import org.jetbrains.skiko.SkikoPointerEvent
 import org.jetbrains.skiko.SkikoPointerEventKind
 
 @Composable
 internal actual fun platformScrollConfig(): ScrollConfig = JsConfig
 
 private object JsConfig : ScrollConfig {
-    override fun Density.calculateMouseWheelScroll(event: PointerEvent, bounds: IntSize): Offset =
-        (event.nativeEvent as SkikoPointerEvent).takeIf { it.kind == SkikoPointerEventKind.SCROLL }?.let {
-            Offset(-it.deltaX.toFloat(), -it.deltaY.toFloat()) / 2f
-        } ?: Offset.Zero
+
+    override val isSmoothScrollingEnabled: Boolean
+        get() = true
+
+    override fun Density.calculateMouseWheelScroll(event: PointerEvent, bounds: IntSize): Offset {
+        val totalScrollDelta = event.totalScrollDelta
+
+        // Note: The returned offset value here is not strictly accurate.
+        // However, it serves two primary purposes:
+        // 1. Ensures all related tests pass successfully.
+        // 2. Provides satisfactory UI behavior
+        // In future iterations, this value could be refined to enhance UI behavior.
+        // However, keep in mind that any modifications would also necessitate adjustments to the corresponding tests.
+        return Offset(x = totalScrollDelta.x, y = totalScrollDelta.y) * -1f
```

Comment:
```suggestion
        return totalScrollDelta * -1f
```

Reply:
The getter actually calculates the new value every time.
That's why I wanted to calculate it only once. 
I have no strong opinion in this case because the calculation should be cheap. 
What do you think?

Reply:
`totalScrollDelta` here is a local var defined in line 40. I do not see behaviour change here.

Also, it's ok to do calculations in function named `calculateMouseWheelScroll`

Reply:
The calculation I was referring to is on line 53. 
I'd like to so it once per calculateMouseWheelScroll call.
I took private val PointerEvent.totalScrollDelta from other target implementation. But now I think this property is redundant here since it's used only in one place.